Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -297,21 +318,21 @@ extension KeyboardShortcuts {
 							"force_use_shortcut".localized
 						]
 					)
-
+					
 					self.focus()
-
+					
 					// If the user has selected "Use Anyway" in the dialog (the second option), we'll continue setting the keyboard shorcut even though it's reserved by the system.
 					guard modalResponse == .alertSecondButtonReturn else {
 						return nil
 					}
 				}
-
+				
 				self.stringValue = "\(shortcut)"
 				self.showsCancelButton = true
-
+				
 				self.saveShortcut(shortcut)
 				self.blur()
-
+				
```

Comment:
Don't make unrelated changes.

Reply:
Sorry - I'll try not to mess up more formatting